Sources/SATSCore/Assets/ThemeColors.xcassets/Gold Theme/Background Top/goldGradientEnd.colorset/Contents.json

```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0.337",
-          "green" : "0.302",
-          "red" : "0.271"
+          "blue" : "0x24",
```

Comment:
Is this color correct? When I lookup Gradient Gold End (light mode) in the design system i get hex color: 14233C

Reply:
The right color is #262624. Some color names are different that the color values shown

Reply:
Great!